### PR TITLE
Variable verbosity levels on `backtest`

### PR DESCRIPTION
**Pull request by jbdelmundo**
Currently planning to have different levels
`0` - no output

`1` - Strategy level: (How the strategy works) strat parameters, Final PNL, Strategy Summary
Use: Compare different strategies, parameters and their returns

`2` - Transaction level: All transactions are logged
Use: Understand how the strategy is working

`3` - Periodic Logging: For every row(day or time) in the data

### CODE REVIEW

**Comment by jbdelmundo** [comment]
Though I build it on top of https://github.com/enzoampil/fastquant/pull/278

**Comment by enzoampil** [comment]
>Though I build it on top of #278

I notice this merge will edit the notebook. Is this actually your intention?

File: python/fastquant/backtest/backtest.py [hunk 1]
```diff
@@ -118,7 +118,7 @@ def backtest(
     channel : str
         Channel to be used for last day notification - e.g. "slack" (default=None)
     verbose : int
-        Verbose can take values: [-1, 0, 1, 2], with increasing levels of verbosity (default=0).
+        Verbose can take values: [0, 1, 2, 3], with increasing levels of verbosity (default=1).
```
Comment (enzoampil) [inline python/fastquant/backtest/backtest.py:121]: Agree that starting with 0 is more intuitive :smile:

File: python/fastquant/backtest/backtest.py [hunk 2]
```diff
@@ -464,3 +467,41 @@ class CustomData(bt.feeds.PandasData):
...
+        print("\t".join(items))
+
+    if format == "indent":
+        if title != "":
+            print(title)
+        for key, value in d.items():
+            print("\t%s:%s" % (key, value))
+
+
+def get_logging_params(verbose):
```
Comment (enzoampil) [inline python/fastquant/backtest/backtest.py:487]: I love this implementation. A lot cleaner than before, great thinking :smile:

**Review by enzoampil — APPROVED** [review]
@jbdelmundo Thanks so much for this Benj! Looks great :smile:

Looks good to merge for me. I think in a later PR, we can fully switch to utilizing `logging.info` from `print`, but can do this later.

**Pull request merged**